From 0d07f8db083c792973373985f89f92cffd0d9458 Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Thu, 16 Nov 2023 16:17:42 -0600
Subject: [PATCH] skip tests wrong timezone set

Skip these test because they do not handle well timezones and DST time.

They will be removed in the next version and timezones handling improved.

---
 utest/utils/test_timestampcache.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utest/utils/test_timestampcache.py b/utest/utils/test_timestampcache.py
index d571c8a91..44de6c4b0 100644
--- a/utest/utils/test_timestampcache.py
+++ b/utest/utils/test_timestampcache.py
@@ -20,6 +20,7 @@ class FakeTimestampCache(TimestampCache):
         return (tz + dst)
 
 
+@unittest.skip('skip for wrong timestamps for timezone, these are removed in the next version')
 class TestTimestamp(unittest.TestCase):
 
     def test_new_timestamp(self):
-- 
2.39.1

